ftw/tagging/profiles/uninstall_plone5/metadata.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I don't think an empty metadata.xml is necessary in an uninstall profile at least (I didn't put one in ftw.globalstatusmessage anyway)